content/pt-br/docs/setup/production-environment/container-runtimes.md

```diff
@@ -0,0 +1,254 @@
+{{< /note >}}
+
+<!-- body -->
+
+## Pré-requisitos de instalação e configuração
+
+As etapas a seguir se aplicam como configurações comuns para nós do Kubernetes no Linux.
+
+Etapas podem ser puladas caso tenha certeza que particularidades de configuração não serão necessárias.
+
+Para mais informações, consulte a [Network Plugin Requirements](/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/#network-plugin-requirements) ou a documentação específica para a execução de seu container.
+
+## Encaminhando IPv4 e permitindo que o iptables observe o tráfego em ponte.
+
+Verifique se o módulo `br_netfilter` está carregado executando `lsmod | grep br_netfilter` no terminal.
+
+Para carregá-lo explicitamente, execute `sudo modprobe br_netfilter`.
+
+A fim que o iptable de um nó Linux visualize corretamente o tráfego em ponte, verifique se `net.bridge.bridge-nf-call-iptables` possui valor 1 configurado no seu `sysctl`. Por exemplo:
+
+```bash
+cat <<EOF | sudo tee /etc/modules-load.d/k8s.conf
+overlay
+br_netfilter
+EOF
+
+sudo modprobe overlay
+sudo modprobe br_netfilter
+
+# Parâmetros sysctl exigidos pela configuração, os parâmetros persistem nas reinicializações
+cat <<EOF | sudo tee /etc/sysctl.d/k8s.conf
+net.bridge.bridge-nf-call-iptables  = 1
+net.bridge.bridge-nf-call-ip6tables = 1
+net.ipv4.ip_forward                 = 1
+EOF
+
+# Aplicar parâmetros sysctl sem reinicializar
+sudo sysctl --system
+```
+
+## Drivers cgroup
+ 
+No sistema linux, {{< glossary_tooltip length="all" text="controle de grupos" term_id="cgroup" >}} são utilizados para restringir recursos que são alocados para os processos.
+ 
+Quando o [systemd](https://www.freedesktop.org/wiki/Software/systemd/) é escolhido como o sistema inicial para a distribuição linux, o processo inicial gera e consome um grupo de controle raiz (`cgroup`) e acts como um gerenciador cgroup. Systemd tem uma forte integração com cgroups e aloca cgroup por systemd unidade. É possível configurar seu agente de execução do container e o kubelet para utilizar o `cgroupfs`. Usando `cgroupfs` ao lado de systemd significa que existem dois diferentes grupos de gerentes de cgroup.
+ 
+ 
+Um único gerenciador de cgroup simplifica a visão de quais recursos estão sendo alocados e por padrão terão uma visão mais consistente dos recursos disponíveis e em uso. Quando houver dois gerenciadores cgroup em um sistema, o resultado serão duas visões desses recursos. Neste contexto, pessoas reportaram casos onde nós configurados para usar `cgroupfs` para o kubelet e o Docker, mas `systemd` para o resto dos processos, tornam-se instáveis em situações de pouco recurso.
+ 
+Alterar as configurações de forma que seu container runtime e o kubelet usem `systemd` como o driver cgroup estabilizou o sistema. Para configurar isto para o Docker, defina `native.cgroupdriver=systemd`.
```

Comment:
container runtime -> needs to be translated
Ps.: container needs to be translated to `contêiner`